parser-typechecker/src/Unison/Builtin/Decls.hs

```diff
@@ -115,6 +129,68 @@ builtinDataDecls = rs1 ++ rs
                      (var "a" `arr` Type.app' (var "Optional") (var "a"))
       )
     ]
+  eith = DataDeclaration
+    Structural
+    ()
+    [v "a", v "b"]
+    [ ( ()
+      , v "Either.Left"
+      , Type.foralls () [v "a", v "b"]
+          (var "a" `arr` Type.apps' (var "Either") [var "a", var "b"])
+      )
+    , ( ()
+      , v "Either.Right"
+      , Type.foralls () [v "a", v "b"]
+          (var "b" `arr` Type.apps' (var "Either") [var "a", var "b"])
+      )
+    ]
+  fmode = DataDeclaration
+    Structural
```

Comment:
@dolio any of the file / IO enums should be `unique` types. You pick the GUID, via `head -c 1024 /dev/urandom | shasum`.